content/en/docs/best-practices/channel-naming.md

```diff
@@ -96,25 +96,37 @@ spec:
   sourceNamespace: my-operators
 ```
 
-The Subscription is providing hints to OLM which are used to determine exactly which version of an operator will get deployed onto the cluster, in this example OLM will look for an operator to deploy that belongs to the candidate channel within a specified catalog source.  
+The [Subscription][subscription] is providing hints to OLM which are used to determine exactly which version of an operator will get deployed onto the cluster, in this example OLM will look for an operator to deploy that belongs to the candidate channel within a specified catalog source.  
 
-Note that exactly which operator version is deployed can depend on more than what you specify in the Subscription.  On initial install, OLM will always attempt to install whatever is the head of the specified channel by default.  Settings within the operator’s CSV also are used by OLM to determine exactly which operator version OLM will deploy or upgrade.
+Note that exactly which operator version is deployed can depend on more than what you specify in the [Subscription][subscription].  On initial install, OLM will always attempt to install whatever is the head of the specified channel by default.  Settings within the operator’s CSV also are used by OLM to determine exactly which operator version OLM will deploy or upgrade.
 
 ## NAMING
 
-Channel names are used to imply what form of upgrade you want to offer for your operator.  For example, you might have an operator that has a candidate or alpha version which is not supported as well as a version where support is offered.
+Channel names are used to imply what form of upgrade you want to offer for your operator. For example, you might have an operator with a candidate release version that is not supported and a version where support is offered.
```

Comment:
Do you think it'll be helpful to clarify this point a little more with examples? 
```suggestion
Channel names should imply the nature of official support the user can expect when subscribed to that channel for your operator. For example, the channel name `stable-v2` implies that subscribing to this channel will install an operator with a major version `v2` that you can expect official support for. The channel name `candidate-v2` implies that the user will be installing a candidate release version that does not have official supported.
```

Reply:
The examples are added, I mean, they exist already in the doc